bb_wrapper/models/pagamentos.py

```diff
@@ -86,16 +87,24 @@ def _set_data(cls, values):
             values["email"] = key
         elif key_type == TipoChavePIX.uuid:
             values["identificacaoAleatoria"] = key
-        elif key_type == TipoChavePIX.documento:
-            key_value = cpfcnpj.clear_punctuation(key)
-            if len(key_value) == 11:
-                values["cpf"] = key_value
-            else:
-                values["cnpj"] = key_value
+
+        documento = values.pop("documento")
+        if key_type == TipoChavePIX.documento:
+            cls.verifica_documento(key, values)
+        else:
+            cls.verifica_documento(key=documento, values=values)
```

Comment:
Com a flexibilização do argumento lá no wrapper, é necessário verificar se foi enviado ou não!

```suggestion
        documento = values.pop("documento")
        if document is not None:
            if key_type == TipoChavePIX.documento:
                cls.verifica_documento(key, values)
            else:
                cls.verifica_documento(key=documento, values=values)
```